config/peerpods/podvm/libvirt-podvm-image-cm.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: libvirt-podvm-image-cm
+  namespace: openshift-sandboxed-containers-operator
+data:
+  CLOUD_PROVIDER: "libvirt"
+  # PodVM image distro
+  PODVM_DISTRO: "rhel"
+
+  # Pod VM sources
+  # If changing the source, then ensure the respective payload binaries are available
+  # for the new source
+  CAA_SRC: "https://github.com/confidential-containers/cloud-api-adaptor"
+  CAA_REF: "v0.8.2"
+
+  # Booleans
+  DOWNLOAD_SOURCES: "no"
+  CONFIDENTIAL_COMPUTE_ENABLED: "no"
+
+  # Libvirt specific
+  ORG_ID: ""
+  ACTIVATION_KEY: ""
+  YQ_VERSION : "v4.35.1"
+  YQ_CHECKSUM: "sha256:4e6324d08630e7df733894a11830412a43703682d65a76f1fc925aac08268a45"
+  ARCH: "s390x"
+  GO_VERSION: "1.21.9"
+  PROTOC_VERSION: "26.1"
+  PODVM_TAG: "v1.0"
+  PACKER_VERSION: "v1.9.4"
+  OS_VERSION: "9.2"
```

Comment:
I don't think PROTOC and PACKER version is needed here. PROTOC will be used for binary builds and will be part of payload. PACKER version is not configurable as we need to stick to 1.9.4

Reply:
Also is OS_VERSION the base OS version? if yes, then may be the var can be called as BASE_OS_VERSION or similar ?

Reply:
Is PROTOC_VERSION needed? AFAIU we never build binaries at runtime no?

Reply:
Right

Reply:
Yes not needed, removed